cli/cmd/common.go

```diff
@@ -34,6 +35,8 @@ var (
 	defaultGenpolicySettings []byte
 	//go:embed assets/genpolicy-rules.rego
 	defaultRules []byte
+	// This value is injected at build time.
+	launchDigest = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
This should be taken from the manifest, shouldn't it?

Reply:
In the `generate` command, we don't have a manifest yet, so we need to embed this somewhere. We could use the embedded value from the manifest module and make this public, so we have a centralized place where this value comes from.

Reply:
I think this might be a good idea. What do you think @katexochen?